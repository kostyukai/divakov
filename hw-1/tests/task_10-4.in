16 Z 16.5
