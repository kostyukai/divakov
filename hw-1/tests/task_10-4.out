16Z16.500000
